infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/StreamsSnapshotWriter.java

```diff
@@ -155,9 +155,34 @@ private void processUpdatesShadowStream(List<SMREntry> smrEntries, Long currentS
                                             UUID snapshotSyncId) {
         CorfuStoreMetadata.Timestamp timestamp;
 
+        int size = 0;
+        for (SMREntry smrEntry : smrEntries) {
+            size += smrEntry.getSerializedSize();
+        }
+        log.info("Total Size of Data = {}", size);
```

Comment:
Are we planning on checking the size on the receiver side even in the final cut?

Reply:
No not checking.  But we will have to keep a buffer because final serialization before commit can add some bytes to what we compute.  The above was just for debugging and I will remove it.